### DISCUSSION TITLE
Determine Velocity of Objects, mostly cars

### DISCUSSION THREAD

**Discussion by williamhart-az** [blakeblackshear/frigate#4121] [Ideas]
Have cars zooming past my house and wondered, with some fudge factors, could you determine their velocity?

If so, it would be a good presentation to our city management for the addition of speed bumps or other traffic calming techniques.

Is there existing feature set to implement as a hack?

**Comment by NickM-27** [comment]
I believe there is an existing feature request for this. You could maybe get an approximation but I don't think there are existing feature to do this within frigate.
